AthleteIO/src/ro/siit/Athlete/AthleteRepository.java

```diff
@@ -0,0 +1,129 @@
+package ro.siit.Athlete;
+
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class AthleteRepository {
+
+    private static List<Athlete> athleteList = new ArrayList<>();
+
+    /**
+     * This method reads a file, transfers the data to create objects of type Athlete and adds them to a collection of type ArrayList.
+     *
+     * @param fileName
+     * @return a collection of type ArrayList of objects of type Athlete.
+     * @throws IOException if file not found
+     */
+
+    public static List readFromFileToList(String fileName) throws IOException {
```

Comment:
Metoda nu ar fi trebuit sa fie statica, e si mai ciudat ca metoda are doua efecte secundare: adauga obiecte in lista athleteList , dar si returneaza aceeasi lista. Daca mergi pe ideea de repository, atunci metoda ar fi trebuit sa se numeasca `loadAthelesFromFile`, ar fi trebuit sa incarce atletii in lista `athleteList` si sa returneze `void`. Apoi campul privat `athleteList` l-ai fi putut accesa cu un getter.